형호/Week2_ReverseArr.js

```diff
@@ -0,0 +1,7 @@
+function solution(my_string) {
+    let answer = "";
+    my_string.split("").forEach(function (a) {
+        answer = a + answer;
```

Comment:
해당 부분은
```suggestion
        answer += a;
```
로 쓰는게 더 짧고 간단하겠죠?

Reply:
이게 제가 찾아봤을때 +=로 작성하면 a값이 answer뒤에 더해져서 +=으로 하면, 역순이 아니라 원래식으로 작성될거같아서 일부러 answer = a + answer;로 작성했습니다!

Reply:
아 역순으로 넣고싶어서 그러셨던거군요

아이디어 좋네요!